config/config.go

```diff
@@ -12,12 +12,18 @@ type Config struct {
 	GracefulShutdownTimeout    time.Duration `envconfig:"GRACEFUL_SHUTDOWN_TIMEOUT"`
 	HealthCheckInterval        time.Duration `envconfig:"HEALTHCHECK_INTERVAL"`
 	HealthCheckCriticalTimeout time.Duration `envconfig:"HEALTHCHECK_CRITICAL_TIMEOUT"`
+	VaultToken                 string        `envconfig:"VAULT_TOKEN"`
```

Comment:
Is it worth adding a `json omit` tag to this field? I don't think we would ever want to log this and it would protect against anyone accidentally adding this to log output.

Reply:
good point, thanks!